[AY1920S1-CS2113-F10-4] Gazeeebo

JasonLeeWeiHern opened the pull request:
@yueyuu
@jessteoxizhi
@mononokehime14
@e0323290

langyanbin0314 commented on src/main/java/Duke.java:
```diff
@@ -1,10 +1,63 @@
+import java.util.ArrayList;
+import java.util.Stack;
+
 public class Duke {
```
should be your own product

langyanbin0314 commented on src/main/java/gazeeebo/TriviaManager/TriviaManager.java:
```diff
@@ -0,0 +1,42 @@
+    public static Map<String, ArrayList<String>> CommandMemory;
+//    static {
+//        CommandMemory = new HashMap<>();
+//    }
```
remove unnecessary code

langyanbin0314 commented on src/main/java/gazeeebo/UI/Ui.java:
```diff
@@ -0,0 +1,150 @@
+            readCommand();
+            ArrayList<StringBuilder> password_list;
+            Storage store = new Storage();
+            password_list = store.Password();
```
use camel case for variable

langyanbin0314 commented on src/main/java/gazeeebo/UI/Ui.java:
```diff
@@ -0,0 +1,150 @@
+            password_list = store.Password();
+            if (fullCommand.equals(password_list.get(0).toString())) {
+                System.out.println(welcomemessage);
+                LocalDate a = LocalDate.now();
```
use meaningful variable name

langyanbin0314 commented on src/main/java/gazeeebo/UI/Ui.java:
```diff
@@ -0,0 +1,150 @@
+        }
+    }
+
+    public void UpcomingTask(ArrayList<Task> list) throws ParseException {
```
camel case for function name

langyanbin0314 commented on src/main/java/gazeeebo/commands/Contact/ContactsCommand.java:
```diff
@@ -0,0 +1,64 @@
+            storage.Storages_Contact(toStore);
+            System.out.println("What do you want to do next ?");
+            ui.readCommand();
+        }
```
should not have magic string

langyanbin0314 commented on src/main/java/gazeeebo/commands/Edit/EditBothCommand.java:
```diff
@@ -0,0 +1,60 @@
+            System.out.print("Ok, we have edited your Event description and time." + "\n\tFrom: " + list.get(listIndex).listFormat() + "\n\tTo:   " + newEvent.listFormat() + "\n");
+            list.set(listIndex, newEvent);
+        } else if (breakListWords[0].equals("DA")) {
+
```
avoid empty function body

langyanbin0314 commented on src/main/java/gazeeebo/commands/Edit/EditBothCommand.java:
```diff
@@ -0,0 +1,60 @@
+        System.out.print("Type your description & date:\n");
+        ui.readCommand();
+        String[] breakListWords = list.get(listIndex).toString().split("\\|");
+        if (breakListWords[0].equals("T")) {
```
better use switch case and enum

langyanbin0314 commented on src/main/java/gazeeebo/commands/gpacalculator/ConverterGradetoScoreCommand.java:
```diff
@@ -0,0 +1,31 @@
+        } else if(grade.equals("D")) {
+            score = 1.0;
+        } else if(grade.equals("F")) {
+            score = 0.0;
```
use enum

langyanbin0314 commented on src/main/java/gazeeebo/commands/tasks/TentativeEventCommand.java:
```diff
@@ -0,0 +1,66 @@
+import gazeeebo.storage.Storage;
+import gazeeebo.exception.DukeException;
+import java.io.IOException;
+import gazeeebo.tasks.*;
```
avoid import *, only import used classes

langyanbin0314 reviewed (commented):
Generally well done.
Should consider use switch case for methods that check for keywords matching.
pay attention to variable naming and function naming.

janelayh committed:
Added reposense labels

Commit:
Created storage classes for CAP, contact and password

Commit:
modified Junit test and deletecommand

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	src/test/java/GeneralNotePageTest/GeneralNotePageTest.java

yueyuu committed:
create LogCenter + update error message for module page

jessteoxizhi committed:
Merge pull request #284 from yueyuu/new_input_format

create LogCenter + update error message for module page

janelayh committed:
test expense undo command

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

yueyuu committed:
fix bug in NoteStorage

yueyuu committed:
fix bug in NoteStorage

- read from the wrong file
- kept reading from the txt file in the resource folder

yueyuu committed:
checkstyle fixes + add data for testers

yueyuu committed:
Merge pull request #285 from e0323290/master

Did Expense, Tag JUnit tests, added more exceptions for Expenses and Specialization

mononokehime14 committed:
J-unit testing for studyassist done

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main:
  test expense undo command
  create LogCenter + update error message for module page
  Added reposense labels
  commented out specialization test
  made specific format error messages for module page
  added JUnit Tests for Expenses features
  cleanup note page feature
  solve checkstyle issues + add some JUnit test + clean up Note Page feature
  finish changing how user should input general note commands
  changed some input format to make them be on one line
  add more commands to help page
  Edited some checkstyle
  add more help for commands in help page
  fixed number format exception and index out of bounds exception on Done command for tasks and Delete Expenses command

yueyuu committed:
Merge https://github.com/AY1920S1-CS2113T-F10-4/main into new_input_format

# Conflicts:
#	modules.txt
#	src/main/java/gazeeebo/commands/Command.java

janelayh committed:
Merge pull request #286 from mononokehime14/master

J-unit testing for studyassist, solve potential bug in studyassist

yueyuu committed:
minor change

yueyuu committed:
Merge branch 'new_input_format' of https://github.com/yueyuu/main into new_input_format

# Conflicts:
#	NoteDaily.txt
#	src/main/resources/NoteDaily.txt

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

Commit:
JavaDoc and Remove magic strings

yueyuu committed:
Merge pull request #287 from jessteoxizhi/master

JavaDocs and remove magic strings

yueyuu committed:
Merge https://github.com/AY1920S1-CS2113T-F10-4/main into new_input_format

# Conflicts:
#	src/main/java/gazeeebo/help/HelpText.java

janelayh committed:
done categorylist test

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

janelayh committed:
Merge pull request #288 from yueyuu/new_input_format

Fix bug in NoteStorage + fix some checkstyle

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	Save.txt
#	goal.txt
#	modules.txt

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	NoteDaily.txt
#	src/main/java/gazeeebo/help/HelpText.java

yueyuu committed:
add more JUnit for note feature + fix some checkstyle

yueyuu committed:
Merge https://github.com/AY1920S1-CS2113T-F10-4/main into new_input_format

yueyuu committed:
minor change in text file

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	Expenses.txt
#	Save.txt
#	Study_Plan.txt
#	Trivia.txt
#	modules.txt
#	src/main/java/gazeeebo/commands/contact/AddContactCommand.java
#	src/main/java/gazeeebo/parser/ChangePasswordCommandParser.java
#	src/main/java/gazeeebo/parser/ContactCommandParser.java
#	src/main/java/gazeeebo/parser/ParserManager.java
#	src/main/java/gazeeebo/storage/Storage.java
#	src/test/java/contactCommandTest/DeleteContactCommandTest.java
#	src/test/java/contactCommandTest/FindContactsCommandTest.java

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	Expenses.txt
#	Save.txt
#	Study_Plan.txt
#	Trivia.txt
#	modules.txt
#	src/main/java/gazeeebo/commands/contact/AddContactCommand.java
#	src/main/java/gazeeebo/parser/ChangePasswordCommandParser.java
#	src/main/java/gazeeebo/parser/ContactCommandParser.java
#	src/main/java/gazeeebo/parser/ParserManager.java
#	src/main/java/gazeeebo/storage/Storage.java
#	src/test/java/contactCommandTest/DeleteContactCommandTest.java
#	src/test/java/contactCommandTest/FindContactsCommandTest.java

mononokehime14 committed:
Merge pull request #289 from yueyuu/new_input_format

add more JUnit test for note feature + fix some checkstyle

Commit:
doc for commandstack

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	Save.txt
#	Trivia.txt

Commit:
put naming

mononokehime14 committed:
Merge pull request #291 from JasonLeeWeiHern/master

create storages for cap and contact

janelayh committed:
finished java docs and junit test

janelayh committed:
parser changes

janelayh committed:
deleted info inside files

janelayh committed:
delete files info

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	Password.txt
#	goal.txt
#	modules.txt
#	src/main/java/gazeeebo/commands/tasks/DoAfterCommand.java
#	src/main/java/gazeeebo/commands/tasks/FixDurationCommand.java
#	src/main/java/gazeeebo/help/HelpText.java

Commit:
logging

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

yueyuu committed:
add some logging + handle file IOExceptions

yueyuu committed:
Merge pr from Jason

yueyuu committed:
not needed

yueyuu committed:
Update README.md

yueyuu committed:
Delete Ui.png

yueyuu committed:
add updated UI picture

yueyuu committed:
did more logging + error handling

Commit:
added change password to the main page

mononokehime14 committed:
Merge pull request #292 from JasonLeeWeiHern/master

added change password to the main page

janelayh committed:
need to debug spec

Commit:
checkstyle

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

janelayh committed:
no message

Commit:
Change CAP... files to Cap files.

yueyuu committed:
Merge https://github.com/AY1920S1-CS2113T-F10-4/main into new_input_format

# Conflicts:
#	CAP.txt
#	Save.txt
#	Trivia.txt
#	goal.txt
#	src/main/java/gazeeebo/parser/CAPCommandParser.java
#	src/main/java/gazeeebo/parser/ContactCommandParser.java
#	src/main/java/gazeeebo/parser/ParserManager.java
#	src/main/java/gazeeebo/parser/TaskCommandParser.java
#	src/test/java/DoAfterCommandTest.java
#	src/test/java/EditCommandTest/EditBothTest.java
#	src/test/java/EditCommandTest/EditDescriptionTest.java
#	src/test/java/EditCommandTest/EditTimeTest.java
#	src/test/java/FixDurationCommandTest.java
#	src/test/java/RecurringCommandTest.java
#	src/test/java/contactCommandTest/DeleteContactCommandTest.java
#	src/test/java/contactCommandTest/FindContactsCommandTest.java
#	src/test/java/contactCommandTest/ListContactCommandTest.java
#	src/test/java/contactCommandTest/MainContactsCommandTest.java

yueyuu committed:
Merge pull request #293 from JasonLeeWeiHern/master

Change CAP... files to Cap files.

yueyuu committed:
Merge https://github.com/AY1920S1-CS2113T-F10-4/main into new_input_format

# Conflicts:
#	Save.txt

JasonLeeWeiHern committed:
Merge pull request #294 from yueyuu/new_input_format

  Add some logging + handle file IOExceptions

Commit:
Change PlacesPageStorage

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

JasonLeeWeiHern committed:
Merge pull request #295 from jessteoxizhi/master

Checkstyle, logging, PlacesPageStorage

Commit:
changes to help page

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

yueyuu committed:
updated help page

yueyuu committed:
Merge https://github.com/AY1920S1-CS2113T-F10-4/main into new_input_format

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main: (25 commits)
  Change PlacesPageStorage
  Change CAP... files to Cap files.
  checkstyle
  added change password to the main page
  did more logging + error handling
  add updated UI picture
  Delete Ui.png
  Update README.md
  not needed
  Merge pr from Jason
  add some logging + handle file IOExceptions
  logging
  put naming
  doc for commandstack
  Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main
  minor change in text file
  add more JUnit for note feature + fix some checkstyle
  JavaDoc and Remove magic strings
  minor change
  checkstyle fixes + add data for testers
  ...

yueyuu committed:
update help page

yueyuu committed:
Update README.md

yueyuu committed:
Update README.md

JasonLeeWeiHern committed:
Merge pull request #297 from yueyuu/new_input_format

Updated help page

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	Password.txt
#	Save.txt
#	Trivia.txt
#	src/main/java/gazeeebo/parser/ExpenseCommandParser.java
#	src/test/java/contactCommandTest/DeleteContactCommandTest.java
#	src/test/java/contactCommandTest/FindContactsCommandTest.java
#	src/test/java/contactCommandTest/ListContactCommandTest.java
#	src/test/java/contactCommandTest/MainContactsCommandTest.java

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	Trivia.txt
#	goal.txt
#	src/main/java/gazeeebo/commands/tasks/DoAfterCommand.java
#	src/main/java/gazeeebo/help/HelpText.java
#	src/main/java/gazeeebo/storage/Storage.java
#	src/main/java/gazeeebo/tasks/Timebound.java
#	src/test/java/contactCommandTest/FindContactsCommandTest.java
#	src/test/java/contactCommandTest/ListContactCommandTest.java

janelayh committed:
specialization storage

mononokehime14 committed:
Merge pull request #301 from e0323290/master

Specialization Storage done

mononokehime14 committed:
checkstyling

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main:
  specialization storage
  Update README.md
  Update README.md
  update help page
  updated help page
  no message
  need to debug spec
  delete files info
  deleted info inside files
  parser changes
  finished java docs and junit test
  done categorylist test

# Conflicts:
#	src/main/java/gazeeebo/commands/tasks/TagCommand.java
#	src/main/java/gazeeebo/storage/Storage.java

yueyuu committed:
update help page

yueyuu committed:
format help page

JasonLeeWeiHern committed:
Merge pull request #302 from yueyuu/new_input_format

Format help page

mononokehime14 committed:
checkstyling 2.0

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main:
  format help page
  update help page

mononokehime14 committed:
Check Styling 3.0

janelayh committed:
debugged specialization

janelayh committed:
Small changes

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

janelayh committed:
small edit

JasonLeeWeiHern committed:
Merge pull request #303 from e0323290/master

Debugged Specialization

Commit:
Some checkstyle

Commit:
Solve merge conflict

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	src/main/java/gazeeebo/help/HelpText.java
#	src/test/java/contactCommandTest/AddContactCommandTest.java

Commit:
change back list format

Commit:
CAP txt file

Commit:
Input some stuff in txtfile

jessteoxizhi committed:
Merge pull request #304 from JasonLeeWeiHern/master

Added some docs

Commit:
Error handling for places

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main:
  Input some stuff in txtfile
  CAP txt file
  change back list format
  Solve merge conflict
  Some checkstyle
  small edit
  Small changes
  debugged specialization
  changes to help page

# Conflicts:
#	src/main/java/gazeeebo/commands/studyassist/deleteModuleCommand.java

Commit:
checkstyle

mononokehime14 committed:
testing

janelayh committed:
made specialization neater

janelayh committed:
Merge pull request #305 from jessteoxizhi/master

Error handling for places

janelayh committed:
checkstyle

janelayh committed:
log commit

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

mononokehime14 committed:
Merge pull request #306 from e0323290/master

Organize Specialization

mononokehime14 committed:
checkstyle.....

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main:
  log commit
  checkstyle
  made specialization neater
  Error handling for places

# Conflicts:
#	src/main/java/gazeeebo/commands/places/AddPlacesCommand.java

yueyuu committed:
Create diagrams

yueyuu committed:
Delete diagrams

JasonLeeWeiHern committed:
Create draw.io

JasonLeeWeiHern committed:
CAP_CONTACTS_PASSWORD diagrams

JasonLeeWeiHern committed:
Delete draw_io

JasonLeeWeiHern committed:
Update AboutUs.adoc

Commit:
Add logger

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

mononokehime14 committed:
checkstyling....

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main:
  Update AboutUs.adoc
  Delete draw_io
  CAP_CONTACTS_PASSWORD diagrams
  Create draw.io
  Delete diagrams
  Create diagrams

Commit:
Solve some checkstyle errors

jessteoxizhi committed:
Merge pull request #307 from mononokehime14/master

Checkstyle Main almost done, left with CheckstyleTest

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	src/main/java/gazeeebo/storage/Storage.java

mononokehime14 committed:
Merge pull request #309 from JasonLeeWeiHern/master

solved some checkstyle errors

Commit:
checkstyle

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

janelayh committed:
small edits on specialization

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

yueyuu committed:
add diagram files for Help, Note Feature

also added architecture diagram and  commands component diagram

yueyuu committed:
upload PPP

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

Commit:
checkstyle and junit checkstyle

Commit:
refactor

jessteoxizhi committed:
Merge pull request #317 from e0323290/master

Fixed specialization help and command

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

JasonLeeWeiHern committed:
Merge pull request #318 from jessteoxizhi/master

Checkstyle

Commit:
cap file change

yueyuu committed:
Delete [CS2113T-F10-4][Lee Yueyu]PPP.pdf

yueyuu committed:
upload updated PPP

Commit:
checkstyle

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	docs/team/[CS2113T-F10-4][Lee Yueyu]PPP.pdf
#	src/test/java/ContactCommandTest/AddContactCommandTest.java
#	src/test/java/ContactCommandTest/DeleteContactCommandTest.java
#	src/test/java/ContactCommandTest/FindContactsCommandTest.java
#	src/test/java/ContactCommandTest/ListContactCommandTest.java
#	src/test/java/ContactCommandTest/MainContactsCommandTest.java
#	src/test/java/contactCommandTest/AddContactCommandTest.java
#	src/test/java/contactCommandTest/DeleteContactCommandTest.java
#	src/test/java/contactCommandTest/FindContactsCommandTest.java
#	src/test/java/contactCommandTest/ListContactCommandTest.java
#	src/test/java/contactCommandTest/MainContactsCommandTest.java
#	src/test/java/contactcommandtest/AddContactCommandTest.java
#	src/test/java/contactcommandtest/DeleteContactCommandTest.java
#	src/test/java/contactcommandtest/FindContactsCommandTest.java
#	src/test/java/contactcommandtest/ListContactCommandTest.java
#	src/test/java/contactcommandtest/MainContactsCommandTest.java

mononokehime14 committed:
minor

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main:
  upload updated PPP
  Delete [CS2113T-F10-4][Lee Yueyu]PPP.pdf
  refactor
  checkstyle and junit checkstyle
  upload PPP
  add diagram files for Help, Note Feature
  small edits on specialization
  checkstyle
  Solve some checkstyle errors
  Add logger
  checkstyle

janelayh committed:
reposense labels or tests

janelayh committed:
reposense labelling

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

Commit:
Junit test

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

Commit:
checkstyle

Commit:
Put _ in between spacing in contact.txt

jessteoxizhi committed:
Merge pull request #320 from JasonLeeWeiHern/master

Change test package to small letters

Commit:
error in txt file

mononokehime14 committed:
Merge pull request #321 from JasonLeeWeiHern/master

Resolve error in cap.txt

janelayh committed:
added showWelcomeMessage to Expense and Specialization parsers

Commit:
Added boundary for add feature in cap

janelayh committed:
Fixed small bug in Specialization, and added expensestest package

janelayh committed:
Merge pull request #322 from JasonLeeWeiHern/master

Added boundary for add feature in cap

janelayh committed:
no message

Commit:
change findcapcommand

janelayh committed:
no message

Commit:
Junit for undo places and find places

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

janelayh committed:
Updated NoteDaily txt

yueyuu committed:
Merge pull request #323 from jessteoxizhi/master

Junit and checkstyle

Commit:
catch error in edit command

Commit:
catch error in edit command

Commit:
Catch more erros in edit command

Commit:
txt file

jessteoxizhi committed:
Merge pull request #324 from JasonLeeWeiHern/master

Catch errors in edit block

yueyuu committed:
Delete LeeYueyu.adoc

yueyuu committed:
Delete [CS2113T-F10-4][Lee Yueyu]PPP.pdf

yueyuu committed:
Add files via upload

yueyuu committed:
Update AboutUs.adoc

janelayh committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	CAP.txt
#	src/main/resources/Contact.txt
#	src/test/java/TaskCommandTest/RecurringCommandTest.java
#	src/test/java/taskcmdtest/CalendarViewTest.java
#	src/test/java/taskcmdtest/CategoryListCommandTest.java
#	src/test/java/taskcmdtest/DoAfterCommandTest.java
#	src/test/java/taskcmdtest/EventCommandTest.java
#	src/test/java/taskcmdtest/FixDurationCommandTest.java
#	src/test/java/taskcmdtest/RecurringCommandTest.java
#	src/test/java/taskcmdtest/RescheduleCommandTest.java
#	src/test/java/taskcmdtest/SnoozeCommandTest.java
#	src/test/java/taskcmdtest/TentativeEventTest.java
#	src/test/java/taskcmdtest/TimeboundTest.java
#	src/test/java/taskcommandtest/RecurringCommandTest.java
#	src/test/java/taskcommandtest/TagCommandTest.java

yueyuu committed:
Merge pull request #325 from e0323290/master

Changed packages

yueyuu committed:
Merge conflict resolved

yueyuu committed:
Merge https://github.com/AY1920S1-CS2113T-F10-4/main into new_input_format

# Conflicts:
#	src/test/java/taskcmdtest/CategoryListCommandTest.java
#	src/test/java/taskcmdtest/TagCommandTest.java
#	src/test/java/taskcmdtest/TimeboundTest.java

janelayh committed:
fixed expense command parser test

Commit:
edit undocommand

Commit:
edit undocommand

yueyuu committed:
Merge pull request #326 from e0323290/master

Fixed expense command parser test

yueyuu committed:
Merge https://github.com/AY1920S1-CS2113T-F10-4/main into new_input_format

Commit:
PPP

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

yueyuu committed:
add tasks to save.txt

jessteoxizhi committed:
Merge pull request #327 from yueyuu/new_input_format

Add tasks to save.txt

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

yueyuu committed:
Merge pull request #329 from jessteoxizhi/master

PPP and modify undo command

Commit:
txt file

JasonLeeWeiHern committed:
Added Developer Guide

JasonLeeWeiHern committed:
Delete DeveloperGuide.adoc file

Commit:
edit testing and PPP

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

yueyuu committed:
fix checkstyle issues across project

- sort command still needs to fix checkstyle issues

yueyuu committed:
Merge https://github.com/AY1920S1-CS2113T-F10-4/main into new_input_format

yueyuu committed:
Merge pull request #330 from jessteoxizhi/master

edit test and PPP

JasonLeeWeiHern committed:
Delete UserGuide.adoc

yueyuu committed:
Merge https://github.com/AY1920S1-CS2113T-F10-4/main into new_input_format

# Conflicts:
#	Save.txt

JasonLeeWeiHern committed:
User Guide

Commit:
Pull changes

jessteoxizhi committed:
Merge pull request #331 from yueyuu/new_input_format

Fix checkstyle issues across project

JasonLeeWeiHern committed:
DG

JasonLeeWeiHern committed:
UG

JasonLeeWeiHern committed:
delete old dg

JasonLeeWeiHern committed:
delete old ug

Commit:
deletemodulecommand

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main: (40 commits)
  delete old ug
  delete old dg
  UG
  DG
  User Guide
  Delete UserGuide.adoc
  fix checkstyle issues across project
  edit testing and PPP
  Delete DeveloperGuide.adoc file
  Added Developer Guide
  add tasks to save.txt
  PPP
  edit undocommand
  edit undocommand
  fixed expense command parser test
  Merge conflict resolved
  Update AboutUs.adoc
  Add files via upload
  Delete [CS2113T-F10-4][Lee Yueyu]PPP.pdf
  Delete LeeYueyu.adoc
  ...

# Conflicts:
#	src/main/java/gazeeebo/parser/StudyAssistCommandParser.java

yueyuu committed:
Merge pull request #332 from mononokehime14/master

DG diagram

Commit:
weird changes

janelayh committed:
Janel

janelayh committed:
Delete JanelAng.adoc

janelayh committed:
Submitted PPP

janelayh committed:
Delete [AY1920S1-CS2113T-F10-4][Janel Ang]PPP.pdf

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main:
  Delete [AY1920S1-CS2113T-F10-4][Janel Ang]PPP.pdf
  Submitted PPP
  Delete JanelAng.adoc
  Janel

janelayh committed:
Delete JanelAngYeeHuey.pdf

janelayh committed:
Re-submit PPP

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	docs/team/jessteoxizhi.pdf
#	production/duke.main/gazeeebo/commands/studyassist/deleteModuleCommand.class
#	src/main/java/gazeeebo/storage/SpecializationPageStorage.java
#	src/test/java/expensetest/AddExpenseCommandTest.java
#	src/test/java/expensetest/DeleteExpenseCommandTest.java
#	src/test/java/expensetest/ExpenseCommandParserTest.java
#	src/test/java/expensetest/FindExpenseCommandTest.java
#	src/test/java/taskcmdtest/DeadlineCommandTest.java
#	src/test/java/taskcmdtest/DeleteCommandTest.java
#	src/test/java/taskcmdtest/FindCommandTest.java
#	src/test/java/taskcmdtest/ListCommandTest.java
#	src/test/java/taskcmdtest/TagCommandTest.java
#	src/test/java/taskcmdtest/TodoCommandTest.java

janelayh committed:
Added PPP link to portfolio

JasonLeeWeiHern committed:
delete prev ppp

JasonLeeWeiHern committed:
Jason ppp

Commit:
checkstyle

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

yueyuu committed:
Add files via upload

JasonLeeWeiHern committed:
link to ppp

yueyuu committed:
Delete LeeYueyu.pdf

yueyuu committed:
Update AboutUs.adoc

Commit:
merge

Commit:
refactor places test

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

Commit:
PPP

mononokehime14 committed:
Merge pull request #333 from jessteoxizhi/master

PPP and checkstyle

mononokehime14 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

* 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main:
  PPP
  refactor places test
  Update AboutUs.adoc
  Delete LeeYueyu.pdf
  link to ppp
  Add files via upload
  checkstyle
  Jason ppp
  delete prev ppp
  Added PPP link to portfolio
  Re-submit PPP
  Delete JanelAngYeeHuey.pdf

mononokehime14 committed:
ppp

jessteoxizhi committed:
Merge pull request #334 from mononokehime14/master

PPP

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

# Conflicts:
#	docs/team/jessteoxizhi.pdf

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

yueyuu committed:
Merge pull request #335 from JasonLeeWeiHern/master

Catch error in edit command

Commit:
Edit command error solve

Commit:
Merge branch 'master' of https://github.com/AY1920S1-CS2113T-F10-4/main

janelayh committed:
Delete JanelAngYeeHuey.pdf

janelayh committed:
Re-submitted PPP

janelayh committed:
Delete [AY1920S1-CS2113T-F10-4][Janel Ang]PPP.pdf

yueyuu committed:
Merge pull request #337 from JasonLeeWeiHern/master

Edit command error solved

janelayh committed:
Re-submitted PPP

janelayh committed:
Delete JanelAngYeeHuey.pdf

janelayh committed:
Re-submit PPP

JasonLeeWeiHern committed:
wrong ppp

JasonLeeWeiHern committed:
new ppp
